packages/utils/src/collections.ts

```diff
@@ -91,9 +91,17 @@ export function linkedListChanges(
   const lessers: string[] = []
   const greaters: string[] = []
   for (const it of changeList) {
-    const { lesser, greater } = _linkedListChange(sortedList, it)
+    const { lesser, greater } = _linkedListChange(list, it)
```

Comment:
consider doing `[].contact(sortedList)` since imo it's more idiomatic. and maybe renaming variable to `listClone` or something